src/test_config.js

```diff
@@ -1,15 +1,16 @@
+require('dotenv').config()
 // Modify this file to run index.js locally and not as a GitHub Action.
 
 module.exports = {
   // These parameters are required.
-  GITHUB_TOKEN: null,
-  GITHUB_REPO: null,  // Form of '<owner>/<repo_name>'
-  LEETCODE_CSRF_TOKEN: null,
-  LEETCODE_SESSION: null,
+  GITHUB_TOKEN: process.env.GITHUB_TOKEN,
+  GITHUB_REPO: process.env.GITHUB_REPO,  // Form of '<owner>/<repo_name>'
+  LEETCODE_CSRF_TOKEN: process.env.LEETCODE_CSRF_TOKEN,
+  LEETCODE_SESSION: process.env.LEETCODE_SESSION,
 
   // These parameters are optional and have default values if needed.
-  FILTER_DUPLICATE_SECS: 86400,
-  DESTINATION_FOLDER: null,
-  VERBOSE: true,
-  COMMIT_HEADER: 'Sync LeetCode submission'
+  FILTER_DUPLICATE_SECS: process.env.FILTER_DUPLICATE_SECS || 86400,
+  DESTINATION_FOLDER: process.env.DESTINATION_FOLDER || '',
+  VERBOSE: process.env.VERBOSE || true,
```

Comment:
I believe if process.env.VERBOSE === false, this would still turn into true with `||`, I think `??` would work better for all of these. I'll change it on my side since I'm about to update the code a bit and don't want you to have to deal with the merge conflicts.